### ISSUE TITLE
Organization list doesn't update 

### ISSUE THREAD

**Issue by pranshugupta54** [PalisadoesFoundation/talawa-admin#1881]
**Describe the bug**
If an existing admin becomes an admin for another organization, they need to log out and log back in to view the updated list.

**To Reproduce**
Assumption: Have an admin account that is admin for some organizations and also a user for some organizations.

1. Log in via Admin on browser 1.
2. Log in via Superadmin on browser 2.
3. Update the admin's role for some organization from user to admin.
4. Now check browser 1; you won't see the organization for which you were newly made an admin.

**Expected behavior**
Refreshing should show the updated list of organizations for which the user is an admin.

**Actual behavior**
Only old organizations are listed.

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

**Comment by Alfredoeb9** [comment]
Hello 👋, I'd like to take this on please.

**Comment by pranshugupta54** [comment]
Hey @Alfredoeb9, were you able to re-produce the same now? It's an old issue - gotta check if it's still an error

**Comment by Alfredoeb9** [comment]
Yes was able to reproduce looks like on the OrgList Screen, the problem is it checks against the localStorage but localStorage only gets updated on the login screen.

**Comment by pranshugupta54** [comment]
Yeah so we were actually planning to implement state management for the whole instead of localStorage. (We'll do this after GSoC2024 ends, this may introduce major bugs for all contributors)

**Comment by Alfredoeb9** [comment]
Sounds good, that's what I trying for as a solution but didn't find a redux store for such category.

**Comment by sethdivyansh** [comment]
@pranshugupta54 can i work on this issue?

**Comment by SGI-CAPP-AT2** [comment]
@palisadoes @pranshugupta54 ,
Can I work on this issue?

**Comment by SGI-CAPP-AT2** [comment]
I think this issue should not be resolved right now as mentioned in comment https://github.com/PalisadoesFoundation/talawa-admin/issues/1881#issuecomment-2226591601 
Sorry for wasting the time
I will unassign myself
